gateway/crates/gateway-binary/src/args.rs

```diff
@@ -43,9 +121,12 @@ pub struct Args {
     /// to the Grafbase API.
     #[arg(long, short, env = "GRAFBASE_SCHEMA_PATH")]
     pub schema: Option<PathBuf>,
-    /// Set the tracing level
-    #[arg(short, long, default_value_t = 0)]
-    pub trace: u16,
+    /// Set the logging level
+    #[arg(long = "log", env = "GRAFBASE_LOG")]
+    pub log_level: Option<LogLevel>,
+    /// Set the style of log output
+    #[arg(long, env = "GRAFBASE_LOG_STYLE", default_value_t = LogStyle::Ascii)]
```

Comment:
Maybe we could default to LogStyle::Ansi when the terminal is interactive? I don't mind either way though, just a suggestion.

Reply:
great idea!